packages/ember-headless-form/src/template-registry.ts

```diff
@@ -5,5 +5,35 @@
 import type HeadlessFormComponent from './components/headless-form';
 
 export default interface Registry {
+  /**
+   * Headless form component.
```

Comment:
Seems for this to appear in a template when hovering `<HeadlessForm>`, we need to place it here, instead of in front of the class definition (where we have it also).

Reply:
(Thank you! I'll remember this for toucan-core stuff! 🎉  )

Reply:
I did not know this either, thank you for pointing it out!